.changeset/pink-regions-read.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/slang": patch
```

Comment:
I suggest removing this changeset, since that JSON file is not part of our public API (yet), so we don't maintain versioning/changelogs for it.

Reply:
Done